crates/holochain/tests/regression/must_get_agent_activity_saturation.rs

```diff
@@ -0,0 +1,50 @@
+use holo_hash::ActionHash;
+use holochain::sweettest::{SweetConductorBatch, SweetConductorConfig, SweetDnaFile};
+use holochain_wasm_test_utils::TestWasm;
+use holochain_zome_types::prelude::Record;
+use rand::{thread_rng, Rng};
+use std::time::{Duration, Instant};
+
+// Intended to keep https://github.com/holochain/holochain/issues/3028 fixed.
+// ensure that multiple `must_get_agent_activity` calls do not oversaturate the
+// fetch pool and bring gossip to a halt
```

Comment:
Really like references back to the issue for tests that are preventing regressions, nice!